### PR TITLE
Add better support for multiple flavors

### PR DESCRIPTION
**Pull request by marek-siemieniuk-morawski**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -16,15 +23,15 @@ const networkName = BTCV_NETWORK || defaultNetworkName;
...
 export default {
-  sentryDsn: SENTRY_DSN || '',
+  environment: ENVIRONMENT,
+  isBeta: IS_BETA === 'true',
+  applicationId: APP_ID,
+  applicationName: APPLICATION_NAME,
   hosts,
   port: PORT || '443',
   network: bitcoin.alt_networks[networkName],
-  networkName,
```
Comment (KedziaPawel) [inline config/index.js:23]: networkName is needed

File: src/state/wallets/sagas.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ export function* loadWalletsSaga() {
         i18n.send.error.title,
         i18n.formatString(i18n.wallets.errors.wrongNetwork, {
           walletNetworkName,
-          appNetworkName: config.networkName,
+          appNetworkName: config.network,
```
Comment (KedziaPawel) [inline src/state/wallets/sagas.ts:58]: should be networkName here

File: config/index.js (hunk 1 above)
Comment (marek-siemieniuk-morawski) [reply config/index.js:23]: Done

File: src/state/wallets/sagas.ts (hunk 2 above)
Comment (marek-siemieniuk-morawski) [reply src/state/wallets/sagas.ts]: Done

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**